lib/board.rb

```diff
@@ -50,8 +57,17 @@ def diagonal_coordinates?(coordinates)
   end
 
   def place(ship, coordinates)
+    @existing_coordinates += coordinates
     coordinates.each do |coordinate|
       @cells[coordinate].place_ship(ship)
     end
+      @existing_coordinates
+  end
+  
+
+  def overlap?(coordinates)
+    coordinates.any? do |coord|
```

Comment:
.any? is able to iterate over two arrays and look for overlap, then return true.

Reply:
So thankful to find that .any? method